src/design-system/components/TextLink/TextLink.tsx

```diff
@@ -15,10 +16,16 @@ export interface TextLinkProps {
  * @description Renders a plain, static text link, designed to be used within a
  * block of text.
  */
-export function TextLink({ children, url, handleLinkPress = Linking.openURL }: TextLinkProps) {
+export function TextLink({ children, url, handleLinkPress }: TextLinkProps) {
+  const openInBrowser = useOpenInBrowser();
+
   const onPressHandler = useCallback(() => {
-    handleLinkPress(url);
-  }, [handleLinkPress, url]);
+    if (handleLinkPress) {
+      handleLinkPress(url);
+    } else {
+      openInBrowser(url);
+    }
+  }, [handleLinkPress, openInBrowser, url]);
```

Comment:
I don't know if we can use a hook in this component or if we should.. I'd check to make sure this works correctly.